### PR TITLE
feature/lib-30

### PR DESCRIPTION
**Pull request by Emilo067**
# Cards Component

## Overview

The `Cards` component is a reusable UI element designed to encapsulate content within a styled card. It is built with React and offers flexibility in terms of content and styling. This component can be easily integrated into various parts of a UI where a card layout is needed.

## Features

- **Flexible Content:** The `Cards` component accepts any React node as children, allowing you to place any type of content inside the card.
- **Customizable Styling:** You can apply custom styles to the `Cards` component by passing a `className` prop, which will be combined with the default styles.
- **Ref Forwarding:** The component uses `forwardRef` to pass a ref to the root `div` element, enabling direct manipulation of the card in the DOM if needed.

## Props

- `children`: The content to be displayed inside the card. This can include text, images, buttons, or any other React components.
- `className` (optional): A string of custom class names to apply additional styling to the card.
- `...rest`: Any other props will be spread onto the root `div` element, allowing for further customization.

## Usage

```tsx
import { Cards } from './Cards';

function App() {
  return (
    <Cards className="custom-card">
      <h1>Title</h1>
      <p>This is some content inside the card.</p>
    </Cards>
  );
}

### CODE REVIEW

File: src/components/ui/cards/cards.module.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 100%;
+  padding: 26px;
```
Comment (Lelik7777) [inline src/components/ui/cards/cards.module.scss]: думаю, что это лишнее. Ширину и падинги будем уже задавать через className(в пропсах это есть) при использовании

File: src/components/ui/cards/cards.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ * @example
+ * // With additional class names:
+ * <Cards className="custom-class">
+ *   <p>Card Content with custom styling</p>
+ * </Cards>
+ */
+
+export const Cards = forwardRef<ElementRef<'div'>, CardProps>(
+  ({ children, className, ...rest }: CardProps, ref) => {
+    const classCard = clsx(s.card, className)
```
Comment (Lelik7777) [inline src/components/ui/cards/cards.tsx]: лучше  classNames для унификации

File: src/components/ui/cards/cards.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import { type ComponentPropsWithoutRef, type ElementRef, forwardRef } from 'react'
+
+import clsx from 'clsx'
+
+import s from './cards.module.scss'
```
Comment (Lelik7777) [inline src/components/ui/cards/cards.tsx]: styles

File: src/components/ui/cards/cards.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+const meta = {
+  component: Cards,
+  tags: ['autodocs'],
+  title: 'Components/Cards',
+} satisfies Meta<typeof Cards>
+
+export default meta
+type Story = StoryObj<typeof Cards>
+export const Default = {
+  args: {
```
Comment (Lelik7777) [inline src/components/ui/cards/cards.stories.tsx:16]: add children

File: src/components/ui/cards/cards.module.scss (hunk 1 above)
Comment (Emilo067) [reply src/components/ui/cards/cards.module.scss]: исправил

File: src/components/ui/cards/cards.tsx (hunk 2 above)
Comment (Emilo067) [reply src/components/ui/cards/cards.tsx]: исправил

File: src/components/ui/cards/cards.tsx (hunk 3 above)
Comment (Emilo067) [reply src/components/ui/cards/cards.tsx]: исправил

File: src/components/ui/cards/cards.stories.tsx (hunk 4 above)
Comment (Emilo067) [reply src/components/ui/cards/cards.stories.tsx:16]: исправил

**Review by Lelik7777 — APPROVED** [review]

**Review by NoName-ForTeam — APPROVED** [review]

**Comment by Lelik7777** [comment]
Эмиль, поменяй имя таски согласно джире. Сразу не заметил))

**Review by EvgeniyaYakunina — APPROVED** [review]

**Pull request merged**